### DISCUSSION TITLE
ANT BMS - Bluetooth connection

### DISCUSSION THREAD

**Discussion by equi2k** [Louisvdw/dbus-serialbattery#563] [New BMS requests]
Hey and thanks for your work!

Is it possible to connect the ANT BMS via Bluetooth?
In my setup without victron i have connected the ANT BMS via Bluetooth to my Raspberry Pi. Nodered reads the data and send it to my MQTT Broker - i think its not a big deal. Please feel free to contact me, if you need the protcoll / script which i use.

regards,
Christoph

**Comment by Louisvdw** [comment]
Hi
We do have an ticket for bluetooth support, but I am not sure when I will get to it. https://github.com/Louisvdw/dbus-serialbattery/issues/13
If you want to share what you have done then perhaps it will be less work, so yes please.

**Comment by equi2k** [comment]
File is attached!
Bluetooth connect:

sudo bluetoothctl
scan on
pair AA:BB:CC:A6:55:34

rfcomm bind /dev/rfcomm3 00:AA:22:01:23:45 (MAC)

[ANT_BMS_BT.txt](https://github.com/Louisvdw/dbus-serialbattery/files/9527739/ANT_BMS_BT.txt)

**Comment by TimGFoley** [comment]
Did you make any progress with this @equi2k ?   Would be great for the newer Renogy batteries where they removed the RS485 ports.

**Comment by AncoDiara** [comment]
Hello Louis,
What can we do to help? I'm also very much interested to see your script be extended with bluetooth readout instead of coms cables. I'm an experienced BMS builder and tester would like to share ideas around safety policies when bluetooth connected batteries disappear momentarily. Which happens more likely than the bms's with comms cable. But 99.9999% of the time this should not be a problem as the system is in a 'safe' voltage zone of the battery. Thus system should continue to function and hope for the battery to become visible again. etc. Also I would like to see active balancers readout on Cerbo GX from JKbms for example or other nice active balancers.

**Comment by mr-manuel** [comment]
You can check https://github.com/Louisvdw/dbus-serialbattery/discussions/485. A lot of things changed with the last version, so now it should also be easiert to add bluetooth support for other BMS.

**Reply by AncoDiara** [reply]
Indeed many thanks all you guys! it's already exciting to read through all the information around JK-bms on bluetooth. Soon I hope to join the testing. I ordered a few more JK-bms's and will test them on Cerbogx and multi RS combinations.

**Comment by jelcke** [comment]
Is there any news on the Bluetooth option for the ANT bms?

**Reply by mr-manuel** [reply]
Unfortunately not, but you are welcome to open a PR for that
